addons/mail/static/src/models/thread_member/thread_member.js

```diff
@@ -0,0 +1,56 @@
+/** @odoo-module **/
+
+import { registerNewModel } from '@mail/model/model_core';
+import { attr, many2one } from '@mail/model/model_field';
+import { insert, unlink, unlinkAll, insertAndReplace } from '@mail/model/model_field_command';
+
+function factory(dependencies) {
+
+    class ThreadMember extends dependencies['mail.model'] {
```

Comment:
don't forget to override `_createRecordLocalId` or insert on the model won't work